src/controllers/dynakube/oneagent/oneagent_reconciler.go

```diff
@@ -149,11 +149,14 @@ func (r *OneAgentReconciler) getDesiredDaemonSet(dkState *status.DynakubeState)
 	return dsDesired, nil
 }
 
+// getPods loads pods managed by the operator
```

Comment:
```suggestion
// getPods loads oneagent pods managed by the operator
```

or just rename the function to `getOneAgentPods` and remove the comment as it clear enough at that point (I prefer this option 😛 )